### ISSUE TITLE
Left 4 Dead 1 script

### ISSUE THREAD

**Issue by lblanchardiii** [dgibbs64/linuxgsm#108]
After many, many months it seems Valve has finally fixed and/or added the dedicated Left 4 Dead server package to SteamCMD. I modified your L4D2 script slightly earlier today to test if it would work with L4D since I prefer to play that over L4D2 and so far, it seems to be working.

There are a few minor things that don't seem to work. Such as it's not accepting the custom named cfg file for some reason. I can continue to work on it to get it right, but I figured I would mention it to you to see if you'd rather do it since it's your script.

Basically the only thing I changed was the a directory from left4dead2 to left4dead, the name of the cfg file to drop the "2" as well as dropping the "2" from all other parts of the script. I also changed the app ID to 222840 since that's the L4D ID.

Waiting on it to finish installing on my laptop so I can test it out.
